--- a/autotest_ksu.sh	2023-12-09 00:53:07.865362902 +0800
+++ b/autotest_ksu.sh	2023-12-09 00:53:19.813362903 +0800
@@ -24,7 +24,7 @@
 A=$(date +%s)
 export USE_CCACHE=1
 make -j$(nproc) -C $(pwd) O=$(pwd)/out CROSS_COMPILE=$BUILD_CROSS_COMPILE CLANG_TRIPLE=$CLANG_TRIPLE CROSS_COMPILE_ARM32=$CROSS_COMPILE_ARM32 \
-    CC=clang LD=ld.lld \
+    CC=clang LD=$HOME/aarch64-linux-android-4.9/bin/aarch64-linux-android-ld.gold \
     -Werror \
     2>&1 | tee build.txt
 B=$(date +%s)
